app/controllers/admin/api/buyers_applications_controller.rb

```diff
@@ -46,7 +48,8 @@ def create
     application = applications.new(user_account: buyer, plan: application_plan, create_origin: "api")
     application.unflattened_attributes = application_params
     application.user_key = params[:user_key] if params[:user_key]
-    application.application_id = params[:application_id] if params[:application_id]
+    application_id_param = params[:application_id]
```

Comment:
This is not fixing the usage of the params.

Reply:
This is code climate issue

Reply:
What code climate issue? Probably a rubocop one right?
Rubocop is a script with an algorithm that helps detect bad practice. Sometimes it does not apply because it is not perfect. In every case, ask yourself if it makes sense to correct those (following good practice). Maybe it is not necessary in this case, aka false positive.

@nidhi-soni1104  we do not know which cop it violates, could you please write more details for the reviewers to understand and help you better?

Reply:
@hallelujah PR is failed due to code climate issue as it says params[:application_id] is repeated twice  so I fixed it.